projects/imu_driver/src/config.c

```diff
@@ -0,0 +1,21 @@
+#include "config.h"
+#include "imu.h"
+
+// IMU settings
+static ImuSettings imu_settings = { .cs = { .port = GPIO_PORT_A, .pin = 4 },
+                                    .mosi = { .port = GPIO_PORT_A, .pin = 7 },
+                                    .miso = { .port = GPIO_PORT_A, .pin = 6 },
+                                    .sclk = { .port = GPIO_PORT_A, .pin = 5 },
+
+                                    .gyro_first_register = 0x22,
```

Comment:
might want to define these in the header and  choose a better name than "first register"